src/translations/locale/locale.en-US.json

```diff
@@ -1,4 +1,27 @@
 {
+  "App.about.us": "About Us",
+  "App.company": "Company",
+  "App.imprint": "Imprint",
+  "App.privacy.policy": "Privacy policy",
+  "App.signup": "Sign up",
+  "App.login": "Login",
+  "App.email": "E-Mail",
+  "App.password": "Passwort",
+  "App.link.about.us": "https://www.everreal.co/en-US/about-us",
+  "App.link.imprint": "https://www.everreal.co/en-US/imprint",
+  "App.link.privacy.policy": "https://www.everreal.co/en-US/privacy-policy",
```

Comment:
Could be in configuration, but I think it's better to determine it programatically. I assume the auth plugin will always be on the same domain at the url `auth.baseUrl`, so then you can easily get `baseUrl`